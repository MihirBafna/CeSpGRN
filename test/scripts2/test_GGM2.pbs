#PBS -N test_GGM2
#PBS -q hive-himem
#PBS -l nodes=1:ppn=4
#PBS -l walltime=36:20:00
#PBS -j oe
#PBS -o test_GGM2.out
#PBS -m abe                            
#PBS -M zzhang834@gatech.edu  

cd $PBS_O_WORKDIR
module load anaconda3
conda activate myenv
python test_GGM2.py