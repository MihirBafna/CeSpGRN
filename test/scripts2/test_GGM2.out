---------------------------------------
Begin PBS Prologue Mon Oct  4 13:45:24 EDT 2021
Job ID:     7024030.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM2
Queue:      hive-himem
End PBS Prologue Mon Oct  4 13:45:24 EDT 2021
---------------------------------------
---------------------------------------
Begin PBS Epilogue Mon Oct  4 13:54:36 EDT 2021
Job ID:     7024030.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM2
Resources:  nodes=1:ppn=4,walltime=36:20:00,neednodes=1:ppn=4
Rsrc Used:  cput=00:36:13,vmem=354626564kb,walltime=00:09:12,mem=293828648kb,energy_used=0
Queue:      hive-himem
Nodes:     
atl1-1-01-018-35.pace.gatech.edu
End PBS Epilogue Mon Oct  4 13:54:36 EDT 2021
---------------------------------------
---------------------------------------
Begin PBS Prologue Mon Oct  4 13:54:59 EDT 2021
Job ID:     7024036.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM2
Queue:      hive-himem
End PBS Prologue Mon Oct  4 13:54:59 EDT 2021
---------------------------------------
=>> PBS: job killed: walltime 130809 exceeded limit 130800
---------------------------------------
Begin PBS Epilogue Wed Oct  6 02:15:17 EDT 2021
Job ID:     7024036.sched-hive.pace.gatech.edu
User ID:    zzhang834
Job name:   test_GGM2
Resources:  nodes=1:ppn=4,walltime=36:20:00,neednodes=1:ppn=4
Rsrc Used:  cput=144:59:25,vmem=354622468kb,walltime=36:20:19,mem=200275972kb,energy_used=0
Queue:      hive-himem
Nodes:     
atl1-1-01-018-35.pace.gatech.edu
End PBS Epilogue Wed Oct  6 02:15:17 EDT 2021
---------------------------------------
